raiden_contracts/contracts/TokenNetwork.sol

```diff
@@ -766,6 +827,31 @@ contract TokenNetwork is Utils {
         signature_address = ECVerify.ecverify(message_hash, non_closing_signature);
     }
 
+    function recoverAddressFromCooperativeSettleSignature(
```

Comment:
Spec question: Shouldn't the format of this signature be in the spec?

Reply:
Yes. I will make a spec PR after this is merged. This was something that we did not discuss before.